internal/server/assign_task.go

```diff
@@ -127,6 +133,12 @@ func AssignTask(client AssignTaskClient, tmpl template.Template) Handler {
 				delete(data.Error.Field, "assigneeId")
 			} else if err != nil {
 				return err
+			} else if lpa != nil && lpa.CaseType == "DIGITAL_LPA" {
+				// redirect
+				SetFlash(w, FlashNotification{
+					Title: "Task assigned",
```

Comment:
I'm not sure. I went by the title of the page, which is "Assign task", rather than reassign. I don't know whether we need to take into account where the user came from.

Reply:
@GemTay Katie confirmed she wants this to be "Task assigned".